healthcheck.go

```diff
@@ -81,27 +81,27 @@ func (cli *Client) healthcheck(ctx context.Context) (code int, err error) {
 
 	resp, err := cli.httpCli.Do(ctx, req)
 	if err != nil {
-		log.Event(nil, "failed to call elasticsearch", logData, log.Error(err))
+		log.Event(nil, "failed to call elasticsearch", log.ERROR, logData, log.Error(err))
 		return 500, err
 	}
 	defer resp.Body.Close()
 
 	logData["http_code"] = resp.StatusCode
 	if resp.StatusCode < http.StatusOK || resp.StatusCode >= 300 {
-		log.Event(nil, "", logData, log.Error(ErrorUnexpectedStatusCode))
+		log.Event(nil, "", logData, log.ERROR, log.Error(ErrorUnexpectedStatusCode))
```

Comment:
As well as context, leave a message for error - maybe `unexpected status code returned in response`